### ISSUE TITLE
Is there a way to add search templates in jest?

### ISSUE THREAD

**Issue by rajivra** [searchbox-io/Jest#240]
I would like to have the ability to add search templates but the Search options do not indicate an ability to do this.

**Comment by kramer** [comment]
No, search templates are not supported atm. We'll be happy to merge any PRs fixing that though :).
